File: app/src/main/java/com/example/testnew/adapter/TestAdapter.java
package com.example.testnew.adapter;

import com.example.common.base.binding.BaseQuickAdapter;
import com.example.common.base.binding.BaseViewBindingHolder;
import com.example.testnew.databinding.ItemTestBinding;
import com.example.testnew.model.TestListModel;

import java.util.List;

/**
 * Created by WangYanBin on 2020/7/13.
 */
public class TestAdapter extends BaseQuickAdapter<TestListModel, ItemTestBinding> {

    public TestAdapter(List<TestListModel> list) {
        super(list);
    }

    @Override
    protected void convert(BaseViewBindingHolder holder, TestListModel item) {
        if (null != item) {
            ItemTestBinding binding = holder.getBinding();
            binding.ivImg.setBackgroundResource(item.getAvatar());
            binding.tvTitle.setText(item.getTitle());
            binding.tvDescribe.setText(item.getDescribe());
        }
    }

}

File: app/src/main/java/com/example/testnew/adapter/TestAdapter.kt
package com.example.testnew.adapter

import com.example.common.base.binding.BaseQuickAdapter
import com.example.common.base.binding.BaseViewBindingHolder
import com.example.testnew.databinding.ItemTestBinding
import com.example.testnew.model.TestListModel

/**
 * Created by WangYanBin on 2020/8/14.
 */
class TestAdapter(list: MutableList<TestListModel>) : BaseQuickAdapter<TestListModel, ItemTestBinding>(list) {

    override fun convert(holder: BaseViewBindingHolder, item: TestListModel?) {
        if (null != item) {
            val binding: ItemTestBinding = holder.getBinding()
            binding.ivImg.setBackgroundResource(item.avatar)
            binding.tvTitle.text = item.title
            binding.tvDescribe.text = item.describe
        }
    }
}